pyproject.toml

```
@@ -45,7 +45,7 @@ dependencies = [
     "requests>=2.18.4,<3.0.0",
     "rich",
     "rich_click",
-    "s3fs>=2023.3.0",
+    "s3fs>=2023.3.0,<2024.3.1",
```

It's better to use `!=` since they might fix that in the next release.
```suggestion
    "s3fs>=2023.3.0,!=2024.3.1",
```

---

Fixed, thank you!